syntax = "proto3";

package StreamService;

service Hello
{
    rpc SayHello (HelloRequest) returns (HelloReply);
}

service Stream {
    rpc ProcessStream(stream StreamRequest) returns (StreamResponse) {}
}

message HelloRequest
{
    string name = 1;
}

message HelloReply
{
    string msg = 1;
}

message StreamRequest {
    string data = 1;
}

message StreamResponse {
    string result = 1;
}
